rules/renaming/tests/Rector/FileWithoutNamespace/PseudoNamespaceToNamespaceRector/Fixture/fixture6.php.inc

```diff
@@ -12,6 +12,13 @@ class Fixture6
     public function test(\PHPUnit_Framework_TestCase $test): \PHPUnit_Framework_TestCase
     {
     }
+
+    /**
+     * @param \DateTime $time
+     */
+    public function testMore(\DateTime $time)
+    {
+    }
```

Comment:
Could you decoupe this to a new test fixture file? Just so the case is isolated

Reply:
Sure, I put it into a new fixture file, this one was just a quick shot to get this running on CI here.